terraform/docs/tagging-guide.md

```diff
@@ -0,0 +1,100 @@
+# Overview
+This document will describe the tagging strategy to be used within Replatforming terraform code which will be used to create resources in the AWS environment infrastructure.
+
+Tagging is required so that cost and resource utilisation can be processed, with an added benefit of AWS console **Tag Key** searching.
+
+# Tagging Table
+
+The following table highlights the tags **Tag Key** that should added to the AWS resources listed in the section "AWS Resources that can be tagged".
+The common column distinguishes between Tags that have been added as part of an default set and represented by **yes** and those which are unique per service represented by **no**
+
+| **Tag Key** | **Tag Value(s)** | **Description** | **Example** | **Common** |
+|--|--|--|--|--|
+| Name | [ServiceName]-[Environment]-[Workspace] | This is the identifiable name of the service. | publisher-test-nadeem | no |
+| chargeable_entity | govuk-publishing-platform-[Environment] | This is required for billing. | govuk-publishing-platform-test | yes |
+| environment | test int stag prod | Environment area to which this belongs. | test | yes |

```

Comment:
(just making these consistent with the names as typically used elsewhere in GOV.UK)
```suggestion
| environment | test integration staging production | Environment area to which this belongs. | test | yes |

```

Reply:
thank you. totally forgot to update the doc